eclair-core/src/main/scala/fr/acinq/eclair/channel/Channel.scala

```diff
@@ -69,8 +69,8 @@ object Channel {
   // since BOLT 1.1, there is a max value for the refund delay of the main commitment tx
   val MAX_TO_SELF_DELAY = CltvExpiryDelta(2016)
 
-  // as a fundee, we will wait that much time for the funding tx to confirm (funder will rely on the funding tx being double-spent)
-  val FUNDING_TIMEOUT_FUNDEE = 5 days
+  // as a fundee, we will wait that many blocks for the funding tx to confirm (funder will rely on the funding tx being double-spent)
+  val FUNDING_TIMEOUT_FUNDEE = 2016
```

Comment:
Okay I have to say it. How about we simply replaced this with:
```suggestion
  val FUNDING_TIMEOUT_FUNDEE = 2 weeks
```

Reply:
That was my first attempt, but then I thought it would be much better to do the right thing and start using block heights here instead of timestamps, it ensures we're really spec compliant (and our previous code was constantly pushing back the timeout after each restart, which I think was unintended).